string-manipulation-2/src/capitalize-words.js

```diff
@@ -0,0 +1,19 @@
+/* exported capitalizeWords */
+
+function capitalizeWords(string) {
```

Comment:
This code works but I'm sure it was pretty hard to write! Here is another approach in which you can split the initial string into an array that makes for an approach with a much less intensive for loop.

```
function capitalizeWords(string) {
  var words = string.split(' ');
  for (var i = 0; i < words.length; i++) {
    words[i] = words[i][0].toUpperCase() + words[i].slice(1).toLowerCase();
  }
  return words.join(' ');
}
```